docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,284 @@
+Z11_SPF1_MULTIPLE_SAME_MODIFIER | ERROR   | ns_ip_list, spf_modifier      | The following SPF version 1 policy contains more than one {spf_modifier} modifier: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_RECURSIVE              | NOTICE  | ns_ip_list, spf_policy        | The following SPF version 1 policy uses "include" or "redirect": {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_REDIRECT_AND_ALL       | WARNING | ns_ip_list, spf_policy        | The following SPF version 1 policy contains both the "all" keyword and a redirect modifier, which will cause "all" to be ignored: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_REDIRECT_NOT_AT_END    | WARNING | ns_ip_list, spf_policy        | The following SPF version 1 policy contains a redirect modifier, but it does not appear at the end of the string: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_ERROR           | ERROR   | ns_ip_list, spf_policy        | The following SPF version 1 policy has a syntax error: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_OK              | INFO    |                               | The *Child Zone*’s SPF version 1 policy has correct syntax.
+Z11_SPF1_TOO_COMPLEX            | ERROR   | ns_ip_list, spf_policy, count | Evaluating the following SPF version 1 policy would require {count} recursive DNS lookups by an SPF validator, exceeding the limit of 10: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF_WRONG_VERSION           | WARNING | ns_ip_list, spf_version       | The *Child Zone* publishes an SPF policy with invalid version number {spf_version}. Policy retrieved from the following name servers: {ns_ip_list}.
+Z11_UNABLE_TO_CHECK_SPF         | ERROR   |                               | Unable to check whether *Child Zone* publishes an SPF policy.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+## Test procedure
+
+In this section and unless otherwise specified below, the term "[DNS Query]"
+follows the specification for DNS queries as specified in [DNS Query and
+Response Defaults]. The handling of the DNS responses on the DNS queries follow,
+unless otherwise specified below, what is specified for [DNS Response] in the
+same specification.
+
+1. Create a [DNS Query] with query type TXT and query name *Child Zone* ("TXT
+   query").
```

Comment:
What about SPF records? Are they just ignored?

Reply:
The SPF resource record type (type 99) is deprecated and should no longer be used to publish SPF policies. The Zone12 test case looks for any data using this specific type and will emit a warning.

Reply:
Fine.